Extending-OSVR/ConfiguringHDKViveTracking.md

```diff
@@ -0,0 +1,260 @@
+		"renderManagerConfig": {
+			"directModeEnabled": true,
+			"directDisplayIndex": 0,
+			"directHighPriorityEnabled": true,
+			"numBuffers": 2,
+			"verticalSyncEnabled": true,
+			"verticalSyncBlockRenderingEnabled": true,
+			"renderOverfillFactor": 1.2,
+			"renderOversampleFactor": 1.0,
+
+			"window": {
+				"title": "OSVR",
+				"fullScreenEnabled": true,
+				"xPosition": 1920,
+				"yPosition": 0
+			},
+
+			"display": {
+				"rotation": 180,
+				"bitsPerColor": 8
+			},
+
+			"prediction": {
+				"enabled": true,
+				"staticDelayMS": 26,
+				"leftEyeDelayMS": 0,
+				"rightEyeDelayMS": 0,
+				"localTimeOverride": true
+			},
+
+			"timeWarp": {
+				"enabled": true,
+				"asynchronous": false,
+				"maxMsBeforeVSync": 5
+			}
+		}
+	},
+
+	"aliases": {
+		"/me/head": {
+			"posttranslate": [0.0, -0.1143, -0.0635],
+			"rotate": {
+				"axis": "x",
+				"degrees": -90
+			},
+			"child": {
+				"changeBasis": {
+					"x": "x",
+					"y": "y",
+					"z": "z"
+				},
+				"child": "/me/puck"
+			}
+		}
+	}
+}
+````
+It's really easy to switch the HDK2 display for any other OSVR HMD -- just swap the name of the display:
+````json
+"display": "displays/OSVR_HDK_2_0.json",
+````
+could be changed to work with the Sensics dSight display:
+````json
+"display": "displays/Sensics_dSight_landscape_1input_sbs.json",
+````
+Reminder that if you're using OSVRTrackerView to visualize tracked devices, only /me/head, /me/hands/left, and /me/hands/right are visualized by default. To see other devices, specify their path in the command line. For example, to launch OSVRTrackerView and see four connected OSVR-Vive devices (Vive HMD, Controllers, Puck), launch OSVRTrackerView with the following command:
+````c
+OSVRTrackerView.exe --pose /me/head /me/hands/left /me/hands/right /me/puck
+````
+However, if we are aliasing /me/head --> /me/puck like in the server config above, we will be able to see the puck tracking by default without this command.
+
+### Using OSVR-Vive with SteamVR-OSVR
+It is possible to use OSVR-Vive to use the puck as a positional tracker for HDK2, and then play SteamVR games via the SteamVR-OSVR driver. We call it going through the rabbit hole twice. This is an importnat use case for HMD manufacturers to quickly test new HMDs with Vive room-scale tracking by simply mounting a puck. We're still working on some issues on this path after the most recent OpenVR/SteamVR changes, but we think you'll need to make sure you have the following settings in C:\Program Files (x86)\Steam\config\steamvr.vrsettings:
+````json
+"steamvr": {
+		"activateMultipleDrivers": true,
+		"requireHmd" : false
+	}
+````
+To try it, run a server with the config file above, and run SteamVR after installing the latest build of [SteamVR-OSVR](https://github.com/OSVR/SteamVR-OSVR). If it doesn't work immediately, run SteamVR Room Setup. We will continue to update these instructions as the prcoess seems to have changed slightly in the latest releases of SteamVR.
```

Comment:
Run an OSVR server with the `steamvr.vrsettings` config file? Not sure I'm following. You might want to clarify this phrase.